bskycard/src/routes/starter-pack.tsx

```diff
@@ -0,0 +1,98 @@
+import assert from 'node:assert'
+
+import React from 'react'
+import {AppBskyGraphDefs, AtUri} from '@atproto/api'
+import resvg from '@resvg/resvg-js'
+import {Express} from 'express'
+import satori from 'satori'
+
+import {
+  StarterPack,
+  STARTERPACK_HEIGHT,
+  STARTERPACK_WIDTH,
+} from '../components/StarterPack.js'
+import {AppContext} from '../context.js'
+import {httpLogger} from '../logger.js'
+import {handler, originVerifyMiddleware} from './util.js'
+
+export default function (ctx: AppContext, app: Express) {
+  return app.get(
+    '/start/:actor/:rkey',
+    originVerifyMiddleware(ctx),
+    handler(async (req, res) => {
+      const {actor, rkey} = req.params
+      const uri = AtUri.make(actor, 'app.bsky.graph.starterpack', rkey)
+      let starterPack: AppBskyGraphDefs.StarterPackView
+      try {
+        const result = await ctx.appviewAgent.api.app.bsky.graph.getStarterPack(
+          {starterPack: uri.toString()},
+        )
+        starterPack = result.data.starterPack
+      } catch (err) {
+        httpLogger.warn(
+          {err, uri: uri.toString()},
+          'could not fetch starter pack',
+        )
+        return res.status(404).end('not found')
+      }
+      const imageEntries = await Promise.all(
+        starterPack.listItemsSample
+          .map(li => li.subject)
+          .concat(starterPack.creator)
+          // has avatar
+          .filter(p => p.avatar)
+          // no bad labels
+          .filter(p => !p.labels.some(l => hideAvatarLabels.has(l.val)))
```

Comment:
What does this look like if we don't have a full set of images 🤔

Reply:
We had some mockups of this are varying stages, it looked pretty good regardless of the count. Ideally we won't ever have less than 8 but...just in case it doesn't look horrible at all.

Reply:
(Assuming we have the same implementation here)

Reply:
Yeah, I tested it with all counts down to 1.  I think it might be nice to have a fallback design for 1-4, where the background tiling also looks a bit repetitive.